spring-xd-spark-streaming/src/main/scala/org/springframework/xd/spark/streaming/examples/SparkScalaWordCount.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.xd.spark.streaming.examples
+
+import java.util.Properties
+import java.lang.String
+
+import org.apache.spark.streaming.StreamingContext._
+import org.apache.spark.streaming.dstream.{DStream, ReceiverInputDStream}
+import org.springframework.xd.spark.streaming.SparkConfig
+import org.springframework.xd.spark.streaming.scala.Processor
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+class SparkScalaWordCount extends Processor[String, (String, Int)] {
```

Comment:
Do you think we should put the examples into parallel "java" and "scala" sub-packages under "examples". Then we don't need to include Scala in the class names. In fact, we might want to keep Spark out of the class names also, just "WordCount" and "Logger".